.travis.yml

```diff
@@ -0,0 +1,21 @@
+sudo: required
+
+language: python
+
+python:
+  - "3.5"
+
+services:
+  - docker
+
+before_install:
+- docker pull postgres:9.4
+- docker pull minio/minio
+- docker-compose -f local/compose.yml -p imgm up -d
+- docker build -f Dockerfile -t local/imagemanager .
+- docker run -d --rm -it --network imgm_default -p 8000:5000 --network-alias image-manager -v $PWD:/usr/src/app local/imagemanager
```

Comment:
this is just so we can use the service for tests, right?
If so, I think we can drop a couple of options - should make things easier to follow:
> docker run -d --network imgm_default -p 8000:5000 local/imagemanager

---
- `--rm` - has no effect (travis will tear the whole thing down anyways
- `-it` - makes no sense: this is used if we want to attach the current terminal into the container (also redirecting stdin/stdout). Also, is overriden by `-d`
- `-v $PWD` we should make sure we're testing *exactly* what's versioned, so why using the custom mount? ("official" deployments won't.)
- `--network-alias` - since this is not used by any other service of the minimal test deployment, and we're running tests from the host, this is not needed (we'll use localhost:8000 in test.py anyways).